src/frontend/react-hiringsystem/src/App.js

```diff
@@ -3,17 +3,21 @@ import Layout from "./components/shared/Layout";
 import { Route, Routes } from "react-router-dom";
 import Login from "./pages/Login";
 import { AuthContextProvider } from "./components/shared/AuthContext";
+import { CreateJobContextProvider } from "./components/shared/JobContext";
+import {EditJobContextProvider} from "./components/shared/EditJobContext";
```

Comment:
Was this import used in your testing phase? I can't seem to find it

Reply:
Yes, indeed, initially I had two separate contexts for the two crud operations (CreateJobContext and EditJobContext). But I ended up merging them into JobContext.